src/machines/java/com/enderio/machines/client/gui/widget/ActivityWidget.java

```diff
@@ -57,7 +58,14 @@ private void renderToolTip(GuiGraphics guiGraphics, int mouseX, int mouseY) {
             if (list.isEmpty()){
                 list = List.of(MachineLang.TOOLTIP_IDLE);
             }
-            guiGraphics.renderTooltip(screen.getMinecraft().font, list, Optional.empty(), mouseX, mouseY);
+            MutableComponent component = Component.empty();
```

Comment:
I'm not sure this is the ideal solution - not a big fan of creating components each time we render.

Looking into #672 and testing myself, I'd be inclined to be looking at what SagMillScreen is doing differently instead (maybe something related to the grinding ball tooltip?) but I don't think this change is the right way to go.

I might suggest removing this fix for now so we can merge the rest of them, and then keep investigating #672 separately.

Everything else is looking good though, thank you.